MvvmCross/Core/Binding/BindingContext/MvxBaseFluentBindingDescription.cs

```diff
@@ -113,6 +114,37 @@ public MvxSourceStepDescription CreateSourceStep(MvxSourceStepDescription inputs
             }
         }
 
+        public class CombinerSourceSpec
+            : ISourceSpec
+        {
+            private readonly bool _useParser;
+            private readonly string[] _properties;
+            private readonly IMvxValueCombiner _combiner;
+
+            public CombinerSourceSpec(IMvxValueCombiner combiner, string[] properties, bool useParser)
+            {
+                _combiner = combiner;
+                _useParser = useParser;
+                _properties = properties;
+            }
+
+            public MvxSourceStepDescription CreateSourceStep(MvxSourceStepDescription inputs)
+            {
+                var innerSteps = _useParser ?
+                    _properties.Select(p => Mvx.Resolve<IMvxBindingDescriptionParser>().ParseSingle(p).Source) :
```

Comment:
Seems like this sentence can be optimized a bit by moving the `Mvx.Resolve<IMvxBindingDescriptionParser>()` outside of the Select call

Reply:
Make a PR or log an issue. This PR is merged and closed 😄